pkg/webui/console/components/events/previews/shared/description-list/description-list.styl

```diff
@@ -42,3 +42,12 @@
   &:after
     content: '>'
 
+.highlight
```

Comment:
There is nothing wrong with this approach and it seems to work as expected, however now we have 2 places where byte like outline is defined (in `<Byte />` and `<DescriptionList />`. If one changes the styling for one component (say the `<Byte />` as it is used in many places in the Console) we can end up with inconsistent styling within the `<Event />` component.

@kschiffer Is there any other way to highlight the `fCnt` value in the events list? Mb by making it bold (similar to how values are rendered in the V2 Console)?

Reply:
I agree with you that we're risking some inconsistencies here. I think we should regard this fix here as a preliminary solution. Let's use this solution now and I'll open another issue about increasing the recognizability of common data points in the event streams in general.